funcs_utils.php

```diff
@@ -349,83 +272,41 @@ function branch_to_checkout($branches, $defaultBranch, $currentBranch, $currentB
 }
 
 /**
- * Uses composer.json to workout the current branch cms major version
- *
- * If composer.json does not exist then it's assumed to be CURRENT_CMS_MAJOR
+ * Works out the current branch cms major version
  */
 function current_branch_cms_major(
     // this param is only used for unit testing
     string $composerJson = ''
 ) {
-    global $MODULE_DIR;
+    global $MODULE_DIR, $GITHUB_REF;
 
-    // Some repositories don't have a valid matching CMS major
-    $ignoreCMSMajor = [
-        '/silverstripe-simple',
-        '/markdown-php-codesniffer',
-    ];
-    foreach ($ignoreCMSMajor as $ignore) {
-        if (strpos($MODULE_DIR, $ignore) !== false) {
-            return CURRENT_CMS_MAJOR;
-        }
+    // This repo matches every major and matches start at the lowest - but we only want the highest stable.
+    if ($GITHUB_REF === 'silverstripe/silverstripe-simple') {
+        return MetaData::HIGHEST_STABLE_CMS_MAJOR;
     }
```

Comment:
markdown-php-codesniffer will get correctly identified - but the theme still needs some help.